front/src/app/components/forms/convention/SignatureActions.tsx

```diff
@@ -24,7 +23,7 @@ const processedDataBySignatoryRole: Record<
 > = {
   beneficiary: {
     fieldName: getConventionFieldName("signatories.beneficiary.signedAt"),
-    signatoryFunction: "bénéficiaire de l'immersion",
+    signatoryFunction: "bénéficiaire ",
```

Comment:
est-ce voulu l'espace ici ?

```suggestion
    signatoryFunction: "bénéficiaire",
```